tests/task/test_cutOutDonutsCwfsTask.py

```diff
@@ -42,57 +42,56 @@ class TestCutOutDonutsCwfsTask(lsst.utils.tests.TestCase):
     @classmethod
     def setUpClass(cls):
         """
-        Run the pipeline only once since it takes a
-        couple minutes with the ISR.
+        Generate donutCatalog needed for task.
         """
 
         moduleDir = getModulePath()
-        testDataDir = os.path.join(moduleDir, "tests", "testData")
-        testPipelineConfigDir = os.path.join(testDataDir, "pipelineConfigs")
-        cls.repoDir = os.path.join(testDataDir, "gen3TestRepo")
-        cls.runName = "run2"
-        # The visit number for the test data
-        cls.visitNum = 4021123106000
+        cls.testDataDir = os.path.join(moduleDir, "tests", "testData")
+        testPipelineConfigDir = os.path.join(cls.testDataDir, "pipelineConfigs")
+        cls.repoDir = os.path.join(cls.testDataDir, "gen3TestRepo")
+        cls.cameraName = "LSSTCam"
```

Comment:
In this test again it doesn't seem that `cameraName` gets actually called anywhere